src/components/TodoFilter.js

```diff
@@ -0,0 +1,13 @@
+const TodoFilter = ({$todoFilter, onFilter}) => {
+    $todoFilter.addEventListener("click", (e) => {
+        $todoFilter.querySelectorAll("a").forEach(aTag => {
+            if(aTag.classList.contains("selected")){
+                aTag.classList.remove("selected");
+            }
```

Comment:
```suggestion
            aTag.classList.toggle("selected", aTag == e.target);
```

`classList.toggle(class, condition)`
2번째 인자가 T면 클래스 추가, F면 클래스 제거하는 메서드길래 응용해봤어요~ 

line 9 도 줄일 수 있을 것 같네요

Reply:
엇 여기 로직은 `toggle` 만 하는 과정이기보다는,

해당하는 모든 `a` 태그를 돌면서 `selected` 가 포함되어 있으면 단순히 해제하고, 
클릭한 타겟(`e.target`) 만 추가해주는 로직이라서 토글이랑은 아주 조금은 다른 로직이에요..!

그래도 `toggle` 함수는 알고있었는데, 두번째 인자로 `condition` 이 쓰일 수 있는지는 몰랐습니다 :) 
나중에 쓸일있으면 너무 유용할 것 같아요! 👍